torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -191,6 +199,20 @@ bool isSupported(Node* node) {
       }
     }
 
+    // Value is only supported if operands are floats.
```

Comment:
Do we need to exclude non-floating point  tensors if they are the first input ?

Reply:
That's a good point, this could be relaxed.